lib/ecto_ldap/adapter.ex

```diff
@@ -27,18 +27,25 @@ defmodule Ecto.Ldap.Adapter do
 
     {:ok, connection} = Exldap.connect
     # IEx.pry
-    {:ok, search_results} = Exldap.search_field(
+
+    search_response = Exldap.search_field(
```

Comment:
This isn't going to work anymore because the `prepare` creates a payload for `:eldap` now, not `Exldap`.